packages/sanity/src/core/store/bundles/reducer.ts

```diff
@@ -94,7 +94,7 @@ export function bundlesReducer(
 
       // only capture the deleted bundle if deleted by another user
       const nextDeletedBundles =
-        !isDeletedByCurrentUser && localDeletedBundle
+        !!isDeletedByCurrentUser && localDeletedBundle
```

Comment:
This was mistakenly merged in the prior change - for testing I disabled the check for only deletions by other users, but this should not be in the production usage